libs/components/data-manager/src/lib/modules/data-manager/data-manager-toolbar/data-manager-toolbar.component.spec.ts

```diff
@@ -807,7 +834,74 @@ describe('SkyDataManagerToolbarComponent', () => {
     expect(dataManagerService.updateDataState).not.toHaveBeenCalled();
   });
 
-  it('should pass accessibility', async () => {
-    await expectAsync(dataManagerToolbarNativeElement).toBeAccessible();
+  describe('a11y', () => {
+    it('should set accessibility labels correctly when no list descriptor is given', async () => {
+      const patchInfoSpy = spyOn(
+        SkyContentInfoProvider.prototype,
+        'patchInfo'
+      ).and.stub();
+
+      spyOn(dataManagerService, 'getViewById').and.returnValue({
+        ...(dataManagerToolbarComponent.activeView as SkyDataViewConfig),
+        multiselectToolbarEnabled: true,
+        columnPickerEnabled: true,
+      });
+      dataManagerToolbarFixture.detectChanges();
+
+      expect(patchInfoSpy.calls.count()).toBe(1);
+      expect(patchInfoSpy.calls.all()[0].args).toEqual([
+        { descriptor: undefined },
+      ]);
+
+      expect(getClearAllButton()?.getAttribute('aria-label')).toBeNull();
+      expect(getColumnPickerButton()?.getAttribute('aria-label')).toBe(
+        'Columns'
+      );
+      expect(getSectionFilterCheckbox()?.getAttribute('aria-label')).toBeNull();
+      expect(getSelectAllButton()?.getAttribute('aria-label')).toBeNull();
+    });
+
+    it('should set accessibility labels correctly when a list descriptor is given', async () => {
+      const patchInfoSpy = spyOn(
+        SkyContentInfoProvider.prototype,
+        'patchInfo'
+      ).and.stub();
+
+      const dataManagerFixture = TestBed.createComponent(
+        DataManagerFixtureComponent
+      );
+      dataManagerFixture.componentInstance.dataManagerConfig.listDescriptor =
+        'constituents';
+      dataManagerFixture.detectChanges();
+
+      expect(patchInfoSpy.calls.count()).toBe(1);
+      expect(patchInfoSpy.calls.all()[0].args).toEqual([
+        { descriptor: 'constituents' },
+      ]);
+
+      spyOn(dataManagerService, 'getViewById').and.returnValue({
+        ...(dataManagerToolbarComponent.activeView as SkyDataViewConfig),
+        multiselectToolbarEnabled: true,
+        columnPickerEnabled: true,
+      });
+      dataManagerToolbarFixture.detectChanges();
+
+      expect(getClearAllButton()?.getAttribute('aria-label')).toBe(
+        'Clear all selected constituents'
```

Comment:
Is it correct that we have "constituents" here and in the next 3 items instead of "{0}"?

Reply:
Yes. The `{0}` is the template that gets pulled in - but this is testing the final product where the `{0}` is replaced with `constituents`.